core/storage/src/relational/fetch.rs

```diff
@@ -594,13 +595,8 @@ impl RelationalStorage {
         let cells: Page<CellTable> = conn
             .fetch_page_by_wrapper(wrapper, &PageRequest::from(pagination.clone()))
             .await?;
-        let mut res = Vec::new();
+        let res = cells.records.iter().map(|r| r.to_owned().into()).collect();
         let next_cursor = build_next_cursor!(cells, pagination);
```

Comment:
Avoid unnecessary to_owned/clone
```suggestion
        let next_cursor = build_next_cursor!(&cells, pagination);
        let res = cells.records.into_iter().map(Into::into).collect();
```